lib/api/bucketPutVersioning.js

```diff
@@ -12,7 +12,9 @@ const { config } = require('../Config');
 
 const externalVersioningErrorMessage = 'We do not currently support putting ' +
 'a versioned object to a location-constraint of type Azure.';
-
+const invalidBucketStateMessage = `A replication configuration is present on
+ this bucket, so you cannot change the versioning state. To change the
+ versioning state, first delete the replication configuration.`;
```

Comment:
Newlines are maintained in backtick strings, might want to use string concats like the variable above.

Reply:
True, I don't know what I was thinking